keps/349-all-or-nothing/README.md

```diff
@@ -225,6 +231,14 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// recoveryTimeoutSeconds defines optional time duration in seconds, relative to
+	// pod's failure expressed by PodsReady=false condition, after a Workload is Admitted and running.
+	// After exceeding the timeout the corresponding job gets suspended again
+	// and moved to the ClusterQueue's inadmissibleWorkloads list. The timeout is
+	// enforced only if waitForPodsReady.enable=true. Defaults to 3 mins.
+	// +optional
+	recoveryTimeoutSeconds *int64 `json:"recoveryTimeoutSeconds,omitempty"`
```

Comment:
I think it inconsistent to have Timeout *metav1.Duration and recoveryTimeoutSeconds *int64

Reply:
That's a good point. However, I believe that in general we want to gravitate towards using ints and names with units in it, as above. It's a good practice since it makes API more versatile

Reply:
Actually I think I would prefer consistency, and human readability, here for global configuration.

 indeed we tend to use units suffix in API exposed to clients, but for configuration API this is mostly manual by admin. 

the reason to avoid the Duration syntax in client facing clients is that some clients in other languages than go may not know the syntax

Reply:
I see, in that case let's make it consistent with the rest of the API